apps/trustlab/src/app/preview/route.ts

```diff
@@ -50,14 +51,22 @@ export async function GET(
 
   const draft = await draftMode();
 
-  if (!user) {
+  if (!authResult) {
     draft.disable();
     return new Response("You are not allowed to preview this page", {
       status: 403,
     });
   }
 
-  //TODO: Add further checks to ensure the user is allowed to preview this page
+  const user = authResult.user || authResult;
+
+  const isEditor = canManageContent(user);
```

Comment:
```suggestion
  const user = authResult.user || authResult;
  const isEditor = canManageContent(user);
```
Why empty lines between related code?